### ISSUE TITLE
[Feature Request]: Feature to define packages that you don't want to install with chaotic-aur in the custom_hypr.lst and custom_apps.lst

### ISSUE THREAD

**Issue by sitolam** [prasanthrangan/hyprdots#2070]
### Description

A feature where you add for example `nochoatic:` before a package in `custom_hypr.lst` or `custom_apps.lst` to indicate that the package shouldn't be installed with choatic-aur.

### Use Case

This is usefull because some packages won't install with chaotic-aur.

**Comment by kRHYME7** [comment]
Nope, we don't use chaotic aur in hyprdots. also

> This is useful because some packages won't install with chaotic-aur.

can you explain this further?? Install breaks using chaotic aur?????

**Comment by sitolam** [comment]
> Nope, we don't use chaotic aur in hyprdots.

I installed hyprdots with `Hyde-install` and it installed chaotic-aur. I also find this in the repo of hyde-cli: https://github.com/search?q=repo%3AHyDE-Project%2FHyde-cli%20chaotic&type=code . Maybe I did something wrong. But I like chaotic-aur.

> can you explain this further?? Install breaks using chaotic aur?????

I was trying to install `ags-hyprpanel-git` and it just resulted in errors, and then I installed it without chaotic-aur with this command: `yay -S ags-hyprpanel-git --config /etc/pacman.conf.t2.bkp` because in the backup chaotic-aur is not included and then it installed successfully.
